backend/attendance-service/src/main/java/org/egov/repository/rowmapper/StaffQueryBuilder.java

```diff
@@ -0,0 +1,97 @@
+package org.egov.repository.rowmapper;
+
+import org.apache.commons.lang3.StringUtils;
+import org.egov.web.models.AttendanceRegisterSearchCriteria;
+import org.springframework.stereotype.Component;
+
+import java.util.Collection;
+import java.util.List;
+
+@Component
+public class StaffQueryBuilder {
+
+    private static final String FETCH_ATTENDANCE_REGISTER_QUERY = "SELECT register.*," +
+            "staff.id AS staffId,staff.individual_id AS staffIndId,staff.register_id AS staffRegId,staff.tenantid AS staffTenant,staff.enrollment_date AS staffEnroll,staff.deenrollment_date AS staffDeenroll "+
```

Comment:
Do we require register.* in this search query as we are using this query to show only staff data

Reply:
We also need to send the register details like id, number